src/vs/workbench/api/node/mainThreadEditorsTracker.ts

```diff
@@ -392,9 +388,13 @@ export class MainThreadTextEditor {
 		return false;
 	}
 
-	insertSnippet(template: string, opts: IInsertSnippetOptions) {
+	insertSnippet(template: string, opts: IUndoStopOptions) {
```

Comment:
@jrieken Should `console.warn('applyEdits on invisible editor');` be added to `insertSnippet`?